ad_block_client.cc

```diff
@@ -1472,6 +1495,15 @@ int serializeFilters(char * buffer, size_t bufferSizeAvail,
     }
     bufferSize++;
 
+    if (f->tagLen > 0) {
+      if (buffer) {
+        buffer[bufferSize] = '#';
```

Comment:
Is it safe to assume the buffer has the space available for this additional data? Wasn't quite sure how that side of things works.

Reply:
Yep how it works is it calls the function 2 times. The first with nullptr buffer to get the size. The second it specifies a buffer of that size.

Reply:
OK, got it!